girder/utility/filesystem_assetstore_adapter.py

```diff
@@ -106,6 +107,7 @@ def __init__(self, assetstore):
         # happens to existing assetstores that no longer can access their temp
         # directories.
         self.tempDir = os.path.join(self.assetstore['root'], 'temp')
+        self.deleteLock = threading.Lock()
```

Comment:
Somehow I thought we were creating singletons of the adapters for each assetstore (I don't know why I thought this, since it is wrong).